File: java-app/src/main/java/org/jobrunr/example/services/EmailService.java
package org.jobrunr.example.services;

public class EmailService {

    public void sendConfirmation(String email) {
        IO.println("Confirmation email sent to " + email);
    }

    public void sendWelcome(String email) {
        IO.println("Welcome email sent to " + email);
    }

    public void sendWeeklyDigest() {
        IO.println("Weekly digest sent to all subscribers");
    }
}

File: kotlin-app/src/main/kotlin/org/jobrunr/example/services/EmailService.kt
package org.jobrunr.example.services

class EmailService {

    fun sendConfirmation(email: String) {
        println("Confirmation email sent to $email")
    }

    fun sendWelcome(email: String) {
        println("Welcome email sent to $email")
    }

    fun sendWeeklyDigest() {
        println("Weekly digest sent to all subscribers")
    }
}